build.gradle

```diff
@@ -411,9 +411,11 @@ subprojects { subproj ->
         // Dependencies for logging to cloud storage, as well as the various clients used to do so.
         implementation libs.appender.log4j2
         implementation libs.aws.java.sdk.s3
+        implementation libs.aws.java.sdk.sts
         implementation libs.google.cloud.storage
 
-        implementation libs.s3
+        implementation(platform(libs.aws-bom))
+        implementation libs.bundles.aws
```

Comment:
These deps have been included in build.gradle for both airbytehq/airbyte and airbytehq/airbyte-platform. Do we need them in both places?